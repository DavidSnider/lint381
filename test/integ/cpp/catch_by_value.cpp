catch (foo) {}

catch (bar e) {}

catch (baz&) {}

catch (qux& e) {}

catch (...) {}
