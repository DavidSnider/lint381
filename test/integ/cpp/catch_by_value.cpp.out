catch_by_value.cpp:1:8: error: Catch exceptions by reference, not value
catch (foo) {}
       ^^^
catch_by_value.cpp:3:8: error: Catch exceptions by reference, not value
catch (bar e) {}
       ^^^^^
